backend/domain/VoteHandler.js

```diff
@@ -35,16 +35,17 @@ class VoteHandler {
 
         // Get the player with the most votes. In the event of a tie, we just select the first player to get that number of votes
         let maxVotes = 0;
-        let votedPlayer;
+        let votedPlayer = null;
```

Comment:
@seif-y Null, as we think it might be good to have `null` represent no played has been killed.